server/jetstream.go

```diff
@@ -228,6 +228,22 @@ func (s *Server) enableJetStream(cfg JetStreamConfig) error {
 	return nil
 }
 
+// This will check if we have the a solicited leafnode that shares the system account.
```

Comment:
system account and allow JS.

Reply:
No we don't check for JS here on purpose. Will introduce that concept per account with bridging work.
I do see comment could be worded better though, will fix that.

Reply:
line 237 below checks that raft is not denied. Meaning this checks more than just if the system Account is mapped.
Thought that makes sense to mention in the comment

Reply:
Fair point. Will do.